src/main/drivers/vtx_common.h

```diff
@@ -82,3 +86,4 @@ void vtxCommonSetPitmode(uint8_t onoff);
 bool vtxCommonGetBandChan(uint8_t *pBand, uint8_t *pChan);
 bool vtxCommonGetPowerIndex(uint8_t *pIndex);
 bool vtxCommonGetPitmode(uint8_t *pOnoff);
+bool vtxCommonGetDeviceCapability(vtxDeviceCapability_t *pVtxDeviceCapability);
```

Comment:
I don't generally like to return values in pointers. For these functions it seems the `bool` return value is not used, or just set to `true`. So just return the value. For `vtxCommonGetBandChan` I'd prefer two separate functions `vtxCommonGetBand` and `vtxCommonGetChannel`. And what's the purpose of the `common` prefix? So why not `vtxGetBand` etc?